frontend/device/android/device.lua

```diff
@@ -390,7 +391,15 @@ function Device:canExecuteScript(file)
 end
 
 function Device:isValidPath(path)
-    return android.isPathInsideSandbox(path)
+    -- the fast check
+    if android.isPathInsideSandbox(path) then
+        return true
+    end
+
+    -- the thorough check
+    local real_ext_storage = realpath(android.getExternalStoragePath())
+    local real_path = realpath(path)
+    return real_path:sub(1, #real_ext_storage) == real_ext_storage
```

Comment:
Just for information, what was the issue here, exactly? Symlinks?

Reply:
The problem on Android is that the external storage path is something like `/storage/sdcard1/xxx`. A shortcut to that would be `/sdcard/xxx`. Both of them are checked by luajit-launcher's `isPathInsideSandbox`.
Unfortunately the pathChooser delivers the realpath, which is `/mnt/media_rw/xxx`